packages/squiggle-lang/src/fr/sampleset.ts

```diff
@@ -94,7 +94,7 @@ const baseLibrary = [
     output: "Array",
     definitions: [
       makeDefinition([frSampleSetDist], frArray(frNumber), ([dist]) => {
```

Comment:
tiny style question - but maybe we should remove some of the {return x}, and instead do:
```
      makeDefinition([frSampleSetDist], frArray(frNumber), ([dist]) => dist.samples)
```

Reply:
That's pretty neat. How did you bring up that window?

Reply:
Oh neat, thanks!